spec/02-integration/09-hybrid_mode/05-ocsp_spec.lua

```diff
@@ -0,0 +1,367 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson.safe"
+local pl_file = require "pl.file"
+
+
+local TEST_CONF = helpers.test_conf
+
+
+local function set_ocsp_status(status)
+  local upstream_client = helpers.http_client(helpers.mock_upstream_host, helpers.mock_upstream_port, 5000)
+  local res = assert(upstream_client:get("/set_ocsp?status=" .. status))
+  assert.res_status(200, res)
+  upstream_client:close()
+end
+
+
+for _, strategy in helpers.each_strategy() do
+  describe("cluster_ocsp = on works with #" .. strategy .. " backend", function()
```

Comment:
we can also add a `cluster_ocsp = on` + `OCSP responder errors, DP are disallowed`